test/integration/users.test.js

```diff
@@ -1040,4 +1037,52 @@ describe("Users", function () {
         });
     });
   });
+
+  describe("POST /users/migrate", function () {
+    it("Should return 401 when user is unauthorize", function (done) {
+      chai
+        .request(app)
+        .post("/users/migrate")
+        .end((err, res) => {
+          if (err) {
+            return done();
+          }
+          expect(res).to.have.status(401);
+          expect(res.body.message).to.equal("Unauthenticated User");
+          return done();
+        });
+    });
+    it("Should update the user", async function () {
+      const response1 = await chai
+        .request(app)
+        .post(`/users/migrate`)
```

Comment:
Shouldn't we stub using `sinon` the call to GitHub API before making this request?
Because in case the API is down our tests will start failing.
Let me know what you think
cc: @prakashchoudhary07

Reply:
Sure I'll add `sinon.stub` as discussed.